code/menuui/barracks.cpp

```diff
@@ -478,13 +485,28 @@ void barracks_init_stats(scoring_struct *stats)
 
 	// add the score from kills
 	Assert((Num_stat_lines + 1) < Max_stat_lines);
-	STRCPY1(Stat_labels[Num_stat_lines], XSTR( "Score from kills only:", 1636));
+	STRCPY1(Stat_labels[Num_stat_lines], XSTR("Score from kills only:", 1636));
 	sprintf(Stats[Num_stat_lines], "%d", score_from_kills);
 	Num_stat_lines++;
 
-	for (i=0; i<Num_stat_lines; i++) {
-		font::force_fit_string(Stat_labels[i], Stat_column1_w[gr_screen.res], Barracks_stats_coords[gr_screen.res][BARRACKS_W_COORD]);
-		font::force_fit_string(Stats[i], Stat_column2_w[gr_screen.res], Barracks_stats2_coords[gr_screen.res][BARRACKS_W_COORD]);
+	int w;
+	Calculated_width = 0;
+	Calculated_X2	 = 0;
+	if ((Unicode_text_mode) && (gr_screen.res != GR_640)) {
```

Comment:
The extra parenthesis aren't needed and don't improve the readability of the code. Please remove them.